src/pages/hyeonjooKim/Main/FeedComponent/Feed.scss

```
@@ -0,0 +1,95 @@
+@mixin flex-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@mixin profileImgStyle {
+  width: 36px;
+  height: 36px;
+  border-radius: 50%;
+  overflow: hidden;
+}
+
+.feeds {
+  width: 40%;
+  height: 95%;
+  margin: 0 10px;
+  border: 1px solid rgb(219, 219, 219);
+  border-radius: 5px;
+  margin-top: 2%;
+
+  .writerProfile {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 10%;
+    background-color: white;
+
+    div {
+      @include flex-center;
+      font-weight: bold;
+
+      .profileImgContainer {
+        @include profileImgStyle();
+        border: 1px solid rgb(219, 219, 219);
+        margin: 0 10px;
```

```suggestion
        @include profileImgStyle;
        border: 1px solid rgb(219, 219, 219);
        margin: 0 10px;
```
위에서 사용한 것 처럼 include할 때 소괄호는 생략해도 됩니다!